extension/duckdb_scanner/src/duckdb_scan.cpp

```diff
@@ -143,6 +143,7 @@ void getDuckDBVectorConversionFunc(
     case PhysicalTypeID::INTERVAL: {
         conversion_func = convertDuckDBVectorToVector<interval_t>;
     } break;
+    case PhysicalTypeID::ARRAY:
```

Comment:
Does this mean we can now read `ARRAY` from DuckDB? If so, we should test this in our duckdb scanner extension. @acquamarin can you take a look?